README.md

```diff
@@ -38,13 +38,19 @@ Simple yet powerful way to get your Rails API compliant with [JSON API](http://j
 
 Support:
 
-* Ruby ~> 2.0 with Rails ~> 4.0;
-* Ruby ~> 2.1 with Rails 5.
+* Ruby => 2.0 with Rails ~> 4.0;
```

Comment:
The idea here was to show versions with the [pessimistic operator](https://robots.thoughtbot.com/rubys-pessimistic-operator). But for a better understanding purpose it may be better to write something like: "Ruby 2.x with Rails 4.x" and "Ruby 2.1+ with Rails 5". What do you think?